moxie-staking/schema.graphql

```diff
@@ -0,0 +1,30 @@
+type BlockInfo @entity {
+  id: ID!
+  hash: Bytes!
+  blockNumber: BigInt!
+  timestamp: BigInt!
+}
+type LockInfo @entity {
+  "index"
+  id: ID!
+  "txHash of the transaction"
+  txHash: Bytes!
+  "logIndex of the transaction"
+  logIndex: BigInt!
+  "User's address"
+  user: Bytes!
+  "Subject's token address"
+  subjectToken: Bytes!
+  "Subject's address"
+  subject: Bytes!
+  "Unlock time in seconds"
+  unlockTimeInSec: BigInt!
+  "Amount locked"
+  amount: BigInt!
+  "isBuy"
+  isBuy: Boolean!
+  "Lock period"
+  lockPeriodInSec: BigInt!
+  "created at block information"
+  createdAtBlockInfo: BlockInfo!
```

Comment:
can you put block number directly in this entity. easier for consumption, joins wont be required